cl/recap/tasks.py

```diff
@@ -373,7 +373,14 @@ def process_recap_zip(self, pk):
     mark_pq_status(pq, "", PROCESSING_STATUS.IN_PROGRESS)
 
     logger.info("Processing RECAP zip (debug is: %s): %s", pq.debug, pq)
-    with ZipFile(pq.filepath_local.path, "r") as archive:
+    try:
+        filepath = str(pq.filepath_local.path)
+    except NotImplementedError:
+        # Implemented for testing purposes
+        # Because of a mistmatch between AWS and local testing
+        filepath = pq.filepath_local.name
```

Comment:
Did you check? I went and read [the docs](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.fields.files.FieldFile.name), and it seems like you could probably get away with changing this to `.name` for all uses?

Reply:
🙇